### PR TITLE
Added coding exercise program

### PR DESCRIPTION
**Pull request by tejashrighogare**

### CODE REVIEW

File: armstrongNumber/armstrongNumberProgram.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package armstrongNumber;
+
+public class armstrongNumberProgram {
```
Comment (prashantkawa) [inline armstrongNumber/armstrongNumberProgram.java]: Class name should start with capital letter

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: armstrongNumber/armstrongNumberProgram.java (hunk 1 above)
Comment (tejashrighogare) [reply armstrongNumber/armstrongNumberProgram.java]: Done changes

File: armstrongNumber/armstrongNumberProgram.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package armstrongNumber;
+
+public class ArmstrongNumberProgram {
+    public static void isArmstrong(int number) {
+        int originalNumber, remainder, result = 0;
+        originalNumber = number;
+        while (originalNumber != 0) {
+            remainder = originalNumber % 10;
+            result += remainder * remainder * remainder;
```
Comment (amitsahu8) [inline armstrongNumber/armstrongNumberProgram.java]: why 3 times remainder * remainder * remainder;

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: armstrongNumber/armstrongNumberProgram.java (hunk 2 above)
Comment (tejashrighogare) [reply armstrongNumber/armstrongNumberProgram.java]: Removed 3 times remainder

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**